bin/printStream.py

```diff
@@ -44,17 +47,22 @@ def main():
     parser.add_argument('--group', type=str,
                         help='Globally unique name of the consumer group. '
                         'Consumers in the same group will share messages '
-                        '(i.e., only one consumer will receive a message, as in a queue).')
+                        '(i.e., only one consumer will receive a message, '
+                        'as in a queue).')
     parser.add_argument('--stampDir', type=str,
-                        help='Output directory for writing postage stamp cutout files.')
+                        help='Output directory for writing postage stamp'
+                        'cutout files.')
```

Comment:
If you're going to wrap to 80 characters, I'd suggest going with "indent one level instead of to parenthesis, keep comment together". We're allowed both methods of indenting a parenthetical.